src/modules/job-manager/queue.c

```diff
@@ -78,7 +80,7 @@ static void jobq_destructor (void **item)
     }
 }
 
-static struct jobq *jobq_create (const char *name)
+static struct jobq *jobq_create (const char *name, json_t *value)
```

Comment:
Suggestion: rename parameter `value` to `config` to indicate that it's the queue configuration for the named queue.